keyboards/acheron/apollo/87h/gamma/chconf.h

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2022 QMK
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
```

Comment:
Is this file needed?

Reply:
Removed in [9a67dbd](https://github.com/qmk/qmk_firmware/pull/16817/commits/9a67dbddc06a0e5cd3c7c38ff925ae18c1392e50)